recipes/pyluach/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "pyluach" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4d4ad7a680390d5fe29bf5191939b801fd1b72a85f5c6713038160a265af62f7
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - pyluach
+
+about:
+  home: https://github.com/simlist/pyluach
+  license: MIT
+  license_family: MIT
+  license_file: 
```

Comment:
Please, change it to 
```
license_file: LICENSE.txt
```

Reply:
And could you please raise an issue on this project to also add the license with the package?

Reply:
Sure thing! Here is the issue #8556